index.html

```diff
@@ -3205,43 +3206,51 @@ <h3>Semantic Annotations</h3>
 }
 </pre>
 
+</section>
+
+<section>
+
+    <h3>Annotating Data Schemas</h3>
+
+    <p>
+        In many cases, context extension may be used to annotate pieces of a data schema, to be
+        able to semantically process actual data during an interaction. For example, the internal
+        state of the <a>Thing</a>, semantically described in RDF, can be embedded in the
+        <a>TD Document</a> and pieces of a data schema can be individually annotated as referring to
+        specific parts of that internal state.
+    </p>
+        
     <p>
-        In many cases, context extension may be used to annotate pieces of
-        a data schema, as in the TD snippet below. The example references
-        SSN/SOSA [[vocab-ssn]] and OM 1.8, an ontology for units of measure
-        [[Rijgersberg-et-al-2013]]. Note
-        that object assigned to <code>temperature</code> is a data schema and not
-        the actual temperature property of the physical world object. More details
-        on modeling the physical world can be found in Appendix
+        The TD snippet below uses the well-known SAREF external <a>Vocabulary</a> to
```

Comment:
Please add a link to SAREF.

Reply:
I don't know about the typographic conventions for W3C recs but I assumed a reference should appear only once: the reference for SAREF appears one section above.

Reply:
See cf53ebf4600df3119cdabc5acf1df2b5219199ed.